### PR TITLE
[plugin.video.BlanketFort] 1.0.0

### PR DESCRIPTION
**Pull request by BlanketFortMedia**
### Description

New video add-on for the web series Blanket Fort as well as other podcasts created by Blanket Fort Media.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by learningit** [comment]
There's a fair bit that needs to be done to get this in shape for the kodi.org repo. Firstly please remove all print statements or change them to use xbmc.log(). Please remove the icon and thumbnail inits fron xbmcgui.listitem inits, they are deprecated in Jarvis - please use the setArt() function instead.

I will give a better read tomorrow, but the above changes are needed everywhere i looked at.

File: plugin.video.BlanketFort/_DMsearch.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        req = urllib2.Request(url,None,headers)
+        response = urllib2.urlopen(req)
+        data = response.read()
+        response.close()
+        return data
+# from AddonScriptorde X:\plugin.video.my_music_tv\default.py
+def cache(url, duration=0):
+    cacheFile = os.path.join(cacheDir, (''.join(c for c in unicode(url, 'utf-8') if c not in '/\\:?"*|<>')).strip())
+    if os.path.exists(cacheFile) and duration!=0 and (time.time()-os.path.getmtime(cacheFile) < 60*60*24*duration):
+        fh = xbmcvfs.File(cacheFile, 'r')
```
Comment (learningit) [inline plugin.video.BlanketFort/_DMsearch.py]: please use with open, rather than open - close

**Comment by learningit** [comment]
You must remove the genesis resolvers. They are banned. There's a lot of resolvers in addition tyo that, do you actually make use of them?

**Comment by learningit** [comment]
please explain whast you are using simple downloader for.

**Comment by BlanketFortMedia** [comment]
I used a template that had them in there. I will remove most of the resolvers as we only use archive. I am new to this but am willing to learn.

**Comment by learningit** [comment]
I also question whether you need all the cache stuff.

On Dec 31, 2016 10:08 PM, "BF Media" <notifications@github.com> wrote:

I used a template that had them in there. I will remove most of the
resolvers as we only use archive. I am new to this but am willing to learn.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/xbmc/repo-plugins/pull/830#issuecomment-269890644>, or mute
the thread

**Comment by BlanketFortMedia** [comment]
Can I cancel this PR and resubmit once I have someone with more knowledge have a look at it?

**Comment by learningit** [comment]
You can, but there's no need to.

**Pull request closed without merging**